docs/v0.5/en/user-guide/python-scripts/getting-started.md

```diff
@@ -30,7 +30,7 @@ def hello() -> vector[str]:
    return "Hello, GreptimeDB"
 ```
 
-Save it as `hello.py`, then post it by [HTTP API](./function.md#http-api):
+Save it as `hello.py`, then post it by [HTTP API](./define-function.md#http-api):
```

Comment:
```suggestion
Save it as `hello.py`, then post it by [HTTP API](./define-function.md#http-api) or edit it by our dashboard:
```

Reply:
Not sure if we have a dashboard usage doc, if we have, we can link it.

Reply:
There is no dashboard doc of Python Scripts, so  it seems odd to writing `edit it by our dashboard` here.
I suggest not writing the dashboard here and I will write the dashboard doc in another PR.

Reply:
@killme2008 PTAL